CHANGELOG.md

```diff
@@ -1,24 +1,6 @@
 # Change Log
 All notable changes to this project will be documented in this file. This change log follows the conventions of [keepachangelog.com](http://keepachangelog.com/).
 
-## [Unreleased]
+## [0.2.0]
```

Comment:
Could you please add to the top of README a note about this and link to this changelog?